presto-main/src/test/java/com/facebook/presto/operator/scalar/TestDateTimeFunctions.java

```diff
@@ -106,6 +113,7 @@ private TestDateTimeFunctions(Session session)
     {
         super(session);
         this.session = session;
+        TIMESTAMP = isLegacyTimestamp(session) ? LEGACY_TIMESTAMP : NEW_TIMESTAMP;
```

Comment:
This seems like a bad idea as removing that will be trickier again. I don't really understand what would you like to change that cannot be expressed in terms of isLegacyTimestamp?